### PR TITLE
wwwoodall - chapter 1

### PR DESCRIPTION
**Pull request by woodall**
hello friends, 
I am going to be teaching both javascript and jasmine through these snippets. Feel free to following along how you best like. 

_ps - [I'm rebuilding my homepage. what do you think?](http://wwwoodall.com)_

### CODE REVIEW

**Comment by woodall** [comment]
notice that you can leave 'general comment's at this larger level.
_(or more granular comments like so...)_

File: notes.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# notes
```
Comment (woodall) [inline notes.md]: are you familiar with Markdown. 
after a [simple google search](https://www.google.com/search?sourceid=chrome-psyapi2&ion=1&espv=2&es_th=1&ie=UTF-8&q=markdown%20intro&oq=markdown%20intro&aqs=chrome..69i57.4039j0j1) i found the [creator of markdown's site](https://daringfireball.net/projects/markdown/basics), an [interactive demo](http://markdowntutorial.com/lesson/2/), and a [nice tutorial](https://guides.github.com/features/mastering-markdown/) on the subject.

long story short, if you type something like 
```
# this is the title of my granular comment
```
then markdown knows you mean 

# this is the title of my granular comment
